### PR TITLE
Repair error display in ressource form

### PR DESCRIPTION
**Pull request by fpagnoux**
May be connected to #196.

Bug: When users leave a field blank, it is impossible to validate, but no error shows.

image

### CODE REVIEW

File: app/js/controllers/foyer/ressources/ressources.js [hunk 1]
```diff
@@ -93,9 +93,7 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
         };
     });
 
-    $scope.montantInvalide = function(montant) {
```
Comment (fpagnoux) [inline app/js/controllers/foyer/ressources/ressources.js]: @MattiSG, 
I had trouble transmitting this function to this insulated scope of the directive, so as it was not doing much, I just got rid of the function.

File: app/js/directives/captureMontantRessource.js [hunk 2]
```diff
@@ -11,13 +11,15 @@ angular.module('ddsApp').directive('montantRessource', function(SituationService
             ressourceType: '=',
             dateDeValeur: '=',
             index: '=',
-            onGoingLabel: '=?'
+            onGoingLabel: '=?',
+            form: '=',
```
Comment (fpagnoux) [inline app/js/directives/captureMontantRessource.js]: So that the directive knows if the form has been validated.

**Comment by HelaG** [comment]
C'est ce qui explique les retours qu'on a eu "je n'arrive pas à obtenir un résultat" ?

**Comment by fpagnoux** [comment]
Je pense oui.

File: app/js/controllers/foyer/ressources/ressources.js (hunk 1 above)
Comment (MattiSG) [inline app/js/controllers/foyer/ressources/ressources.js]: :+1:

**Comment by MattiSG** [comment]
GTM

**Pull request merged**